godot/src/ui/components/backpack/backpack.gd

```diff
@@ -114,7 +118,7 @@ func show_wearables():
 		var wearable_item = wearable_item_instanceable.instantiate()
 		grid_container_wearables_list.add_child(wearable_item)
 		wearable_item.button_group = items_button_group
-		wearable_item.set_wearable(wearable_data[wearable_id])
+		wearable_item.async_set_wearable(wearable_data[wearable_id])
```

Comment:
is it okay this doesn't have an await?

Reply:
Yes. It was the previous behaviour. Without a `await` it will forget the promise. So multiple `async_set_wearable` can be executed at the same time.

Maybe we should add an await. But if I added an await there, I would change the behaviour. So for that case, we must do it in another PR.